trl/core.py

```diff
@@ -123,7 +123,10 @@ def masked_var(values, mask, unbiased=True):
     centered_values = values - mean
     variance = masked_mean(centered_values**2, mask)
     if unbiased:
-        bessel_correction = mask.sum() / (mask.sum() - 1)
+        mask_sum = mask.sum()
+        # note that if mask_sum == 1, then there is a division by zero issue
+        # to avoid it you just need to use a larger minibatch_size
+        bessel_correction = mask_sum / (mask_sum - 1)
```

Comment:
Maybe let's add a warning here in case `mask_sum==1`?

Reply:
I think we should just error out actually — it's gonna create nans anyway. Let's wait for @lvwerra's review and address this comment together

Reply:
Yeah error out sounds good to me here!